11-Backbonejs/Introduction-to-Backbonejs.md

```diff
@@ -40,39 +40,43 @@ There are a lot of opinions about which is better, and oftentimes even opinions
 
 ### Backbone has 5 Components
 1. **Models**: Similar to Rails models, these include the application data and business logic. Models can:
-	- Organize data and business logic.
-	- Load and save data from and to the server.
-	- Emit events when data changes.
+	- Organize data and business logic
+	- Load and save data from and to the server
+	- Emit events when data changes
 1. **Collections**: These are ordered sets of models
 1. **Views**: These render models and collections, and listen for DOM/Model events. Backbone views are really closer to Rails controllers, which has been confusing everyone for years.
-1. **Events**: Events are a module that can be mixed in to any object, giving the object the ability to bind and trigger custom named events.
+1. **Events**: Events are a module, taken from underscore, which can be mixed in to any object, giving the object the ability to bind and trigger custom named events.
 1. **Routers**: Routers let you add paths to the URL so that you can maintain history and allow users to jump to a specific state in your single-page-application.
 
 ![Backbone JS Architecture from http://www.slideshare.net/ronreiter/writing-html5-web-apps-using-backbonejs-and-gae](images/backbonejs-architecture.jpg)
 
 ## Live Code: Task List
-As we learn about Backbone, we'll be using a running example: a task list, similar to the one we built at the beginning of Rails. The difference is, this task list will be a Single Page Application. You can find the initial setup [on GitHub](https://github.com/Ada-C6/backbone-live-code). The initial setup lives on the branch `master`, and the final result that we're working toward lives on `live-code-sequence`.
+As we learn about Backbone, we'll be using a running example: a task list, similar to the one we built at the beginning of Rails. The difference is, this task list will be a Single Page Application. You can find the initial setup linked in the calendar. The initial setup lives on the branch `master`, and the final result that we're working toward lives on the branch `live-code-sequence`.  
+
+It's important to note we'll be refining the project step-by-step implementing functionality and then going back and separating components into backbone Models, Collections & Views so we can neatly separate code by the roles they perform.
+
+You can see the finished version here: [https://tasklist-js.herokuapp.com/](https://tasklist-js.herokuapp.com/) 
 
 The live code setup is a little more complex than what we've done in the past with JavaScript. Instead of directing your browser straight to a file, we'll run a simple webserver, kind of like what we did with Rails. To get started, run the following in the command line:
 
 ```bash
-$ git clone https://github.com/Ada-C6/backbone-live-code.git
+$ git clone https://github.com/Ada-C7/backbone-live-code.git
```

Comment:
Can we add the starting HTML and CSS files to this repo? Just one less thing to do at the beginning.